ngraph/frontend/tensorflow/src/ngraph_conversions.hpp

```diff
@@ -56,7 +54,7 @@ void Transpose3D(std::shared_ptr<ov::Node>& node) {
 
```

Comment:
please remove `Transpose3D` and use just `Transpose`.
Also propose to rename it to `insert_transpose_after`

Reply:
now these transposes functions look ugly

Reply:
fixed in https://github.com/openvinotoolkit/openvino/pull/8477